### PR TITLE
Feature/improve team item

### PR DESCRIPTION
**Pull request by ChristinaPastushchak**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Comment by lhalam** [comment]
24.06s$ pylint --rcfile=.pylintrc *
************* Module team.views
C: 86, 0: Exactly one space required after assignment
                    data['teams'][team]['members_id'][i] ={
                                                         ^ (bad-whitespace)
C: 82,12: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
C: 84,16: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 1]
```diff
@@ -74,34 +103,20 @@ export default class TeamItem extends React.Component {
...
-    }
+    };
     /*changes state when team is deleted*/
     comfirmDeleteTeam = () => {
         this.setState({teamDeleted: true});
-    }
+    };
 
-    render() {
+    render(){
```
Comment (dotiurin) [inline eventually/static/src/containers/teamList/TeamItem.js]: space after "render()" needed

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 2]
```diff
@@ -118,56 +133,92 @@ export default class TeamItem extends React.Component {
...
+                            id={this.props.id}
+                        />
+                    </Card>
+
                 </div>
             );
         } else {
-            teamView = <p>Team '{this.state.name}' successfully deleted</p>;
+            teamView = (
+
```
Comment (dotiurin) [inline eventually/static/src/containers/teamList/TeamItem.js]: extra blank line

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 3]
```diff
@@ -118,56 +133,92 @@ export default class TeamItem extends React.Component {
...
+                            {
+                                this.state.listOfMembers.map(usr => (
+                                    <div key={usr['id']} style={styles.member}>
+                                        <Chip onClick={() => this.props.goToUserProfile(usr['id'])}>
+                                            <Avatar src={(usr['photo']) && getImageUrl(usr['photo'])}/>
+                                            {usr['first_name'] + '  ' + usr['last_name']}
+                                        </Chip>
+                                    </div>
+                                ))
+
```
Comment (dotiurin) [inline eventually/static/src/containers/teamList/TeamItem.js]: extra blank line

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 4]
```diff
@@ -118,56 +133,92 @@ export default class TeamItem extends React.Component {
...
+                            }
+                        </CardText>
+                        <RaisedButton label="Edit" onClick={this.handleOpen} style={styles.button}/>
+                        {delButton}
+                        <div style={styles.footer}></div>
+
```
Comment (dotiurin) [inline eventually/static/src/containers/teamList/TeamItem.js]: extra blank line

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 5]
```diff
@@ -118,56 +133,92 @@ export default class TeamItem extends React.Component {
...
+                            open={this.state.open}
+                            handleClose={this.handleClose}
+                            name={this.props.name}
+                            description={this.props.description}
+                            image={this.props.image}
+                            updateItem={this.props.updateItem}
+                            id={this.props.id}
+                        />
+                    </Card>
+
```
Comment (dotiurin) [inline eventually/static/src/containers/teamList/TeamItem.js]: extra blank line

**Review by VasilHamulyak — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**